charts/kaito/workspace/values.yaml

```diff
@@ -57,4 +59,9 @@ gpu-feature-discovery:
   devicePlugin:
     enabled: false
   gfd:
-    enabled: true
\ No newline at end of file
+    enabled: true
+local-csi-driver:
```

Comment:
You need to set these values as well. They are currently being used in the sync script to disable webhooks and metrics

  --set webhook.enforceEphemeral.enabled=false \
  --set webhook.hyperconverged.enabled=false \
  --set observability.metrics.enabled=false